src/components/Jam.tsx

```diff
@@ -382,6 +399,9 @@ export default function Jam({ wallet }: JamProps) {
                   >
                     <div className="d-flex justify-content-center align-items-center">{t('scheduler.button_stop')}</div>
                   </rb.Button>
+                  {currentSchedule.length === 2 || currentSchedule.length === 8 ? (
```

Comment:
I don't quite understand why it is limited to schedules with certain amounts of entries. Care to elaborate? :pray:

Reply:
Yes, currently the scheduler only supports 3 destination addresses, hence `currentSchedule.length === 8` and,
`currentSchedule === 2` is for the dev mode.

Reply:
The amount of schedule entries depends also on the distribution of funds in your jars
If you want to make it available exclusively in dev mode, please use `isDevMode`.